src/components/ValueTypeDropDown.vue

```diff
@@ -1,17 +1,21 @@
 <template>
-	<select
-		class="querybuilder__dropdown"
-		v-model="selected"
-	>
-		<option
-			class="querybuilder__dropdown-option"
-			v-for="(optionItem, index) in optionItems"
-			:value="optionItem"
-			:key="index"
+	<div class="querybuilder__dropdown">
+		<label class="querybuilder__dropdown-label">Value Type</label>
```

Comment:
I don't think that this will satisfy the test, it probably needs a `for="someId"` attribute to actually link it to the select. _Alternatively_, the select could maybe get an [`aria-labeledby` attribute]9https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute)

Reply:
of course, you're right. I will work on it.

Reply:
Done now.